### ISSUE TITLE
Building and archiving extension

### ISSUE THREAD

**Issue by jmgirven** [fusepilot/create-cep-extension#8]
If I create the basic extension:

    create-cep-extension my-cep-extension

Start it:

    npm start

It works fine, and I can see it as an extension in Adobe. I can see in the built `index.html` that it is simply a redirect to `localhost:3000`, where the react app is being hosted. Now I want to build a production version of the extension and share it with my colleagues:

    npm run build

Add my certificate details to the `.env` file and:

    npm run archive

I then share the .zxp as a private extension through the Adobe exchange. They install it and open it through Window -> Extensions, only to see a blank extension window. Looking at the logs I see that `index.html` is loaded, but it cannot find the JS file:

```
OnResponseStarted: file:///Users/jon/Library/Application%20Support/Adobe/CEP/extensions/my.cep.extension/index.html
OnReadCompleted: "file:///Users/jon/Library/Application%20Support/Adobe/CEP/extensions/my.cep.extension/index.html" bytes_read = 395
OnReadCompleted: "file:///Users/jon/Library/Application%20Support/Adobe/CEP/extensions/my.cep.extension/index.html" bytes_read = 0
ResponseCompleted: file:///Users/jon/Library/Application%20Support/Adobe/CEP/extensions/my.cep.extension/index.html
OnResponseStarted: file:///static/css/main.1f6b6bee.css
ResponseCompleted: file:///static/css/main.1f6b6bee.css
OnResponseStarted: file:///static/js/main.9c1e51de.js
ResponseCompleted: file:///static/js/main.9c1e51de.js
```

The javascript is being loaded from the root directory, which if it were hosted, would be fine. But in the extension resolves to the root file path.

    <script type="text/javascript" src="/static/js/main.9c1e51de.js">

Out of interest, I tried modifying the production build `index.html` to instead have:

    <script type="text/javascript" src="./static/js/main.9c1e51de.js">

I can then see that the JS is loaded correctly, but I experience other issues with the app. As it is a production build, no logs are printed to help me with further issues.

Am I doing something wrong when building or distributing the app? Is Adobe supposed to be serving the build directory rather than opening it as a file?

-------

`npm ls create-cep-extension-scripts`:
create-cep-extension-scripts@1.0.0-beta.10

`node -v`: 
v8.2.1

`npm -v`:
5.3.0

Testing with Adobe Premiere Pro 2017.1.2 v11.0.

**Comment by jmgirven** [comment]
@kennethormandy : Have you experienced this with your fork as well?

**Comment by jmgirven** [comment]
Is it as simple as editing the `package.json` file and setting `"homepage": "."`? See [create-react-app relative paths](https://github.com/facebookincubator/create-react-app/blob/master/packages/react-scripts/template/README.md#building-for-relative-paths).

If so, should this be a default?

**Comment by fusepilot** [comment]
But it works locally? Can you run these commands and try it again locally to see if it still works?

````
defaults write com.adobe.CSXS.7 PlayerDebugMode 0;
defaults write com.adobe.CSXS.6 PlayerDebugMode 0;
defaults write com.adobe.CSXS.5 PlayerDebugMode 0;
defaults write com.adobe.CSXS.4 PlayerDebugMode 0;
````
I haven't tested with Adobe Exchange. It's probably a problem with ZXP signing thats preventing it.

**Comment by fusepilot** [comment]
Closed in 3a3a9c9a3c49cf3db144ccd159d927453cb0e0c7. Let me know if this doesn't fix the problem on your end.
